app/views/charges/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<button type="button" class="btn btn-default"><%= link_to 'New Charge', new_charge_path %></button>
+<p> Charges </p>
+<table class="table table-striped">
+  <th> Amount </th>
+  <th> Created at: </th>
+  <th> Updated at: </th>
+  <th> Description </th>
+  <th> Operation date </th>
+  <th> Actions </th>
+  <% @charges.each do |charge| %>
```

Comment:
This code looks not really pretty. Take a look at *rendering collection* here: http://guides.rubyonrails.org/layouts_and_rendering.html

<%= render partial: "charge", collection: @charges %>